extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -44,10 +48,32 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
 
     /** @param {!ShoppingConfigDataDef} tagData */
     this.tagData_ = null;
+
+    /** @param {!AmpElement} element */
+    this.shoppingAttachment_ = null;
+
+    /** @param {!AmpElement} element */
+    this.shoppingTagEl_ = null;
```

Comment:
Do you need this one?

Reply:
It is used in other parts in this component @processprocess suggested adding it in for clarity. I can remove it if it's not really needed.

Reply:
I will just add a visual diff test instead of a unit test.